examples/remote_cluster.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This example demonstrate communication with a remove Kube cluster from a
+# server outside of the cluster without kube client installed on it.
+# The communication is secured with the use of Bearer token.
+
+from kubernetes import client, config
+
+
+def main():
+    # Define the barer token we are going to use to authenticate.
+    # See here to create the token:
+    # https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/
+    aToken = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+    # Create a configuration object
+    configuration = client.Configuration()
+
+    # Specify the endpoint of your Kube cluster
+    configuration.host = "https://XXX.XXX.XXX.XXX:443"
+
+    # Security part.
+    # In this simple example we are not going to verify the SSL certificate of
+    # the remote cluster (for simplicity reason)
+    configuration.verify_ssl = False
+    # Nevertheless if you want to do it you can with these 2 parameters
+    # configuration.verify_ssl=True
+    # ssl_ca_cert is the filepath to the file that contains the certificate.
+    # configuration.ssl_ca_cert="certificate"
+
+    configuration.api_key = {"authorization": "Bearer " + aToken}
+
+    # Use our configuration
+    client.Configuration.set_default(configuration)
```

Comment:
I suggest you an ApiClient and pass it to client.CoreV1Api instead of changing the default.